app/services/site_channel_verifier.rb

```diff
@@ -87,33 +56,41 @@ def verified_channel_post_verify
       PublisherMailer.verification_done_internal(verified_channel).deliver_later
     end
     SlackMessenger.new(message: "*#{verified_channel.publication_title}* verified by #{verified_channel.publisher.name} (#{verified_channel.publisher.email}); id=#{verified_channel.id}").perform
+
+    # Let eyeshade know about the new Publisher
+    begin
```

Comment:
I don't think we need to wrap it around in `begin` `perform`. If it crashes, my hunch is that we want it to crash loudly.

Reply:
Failures are detected with this code by sending events to Sentry, and we know reasonable fallback solutions:
- Synchronous user initiated verification: Show failure message. Verify again in the future, which happens automatically every hour.
- Async verification: Fail and retry the Eyeshade channel setting. Actually, we don't have a logic path for this -- if this job reruns at this point it will do nothing since channel.verified is true. It would make sense to add this to #842.